### PR TITLE
add msat_total column migration and update prisma

### PR DESCRIPTION
**Pull request by joshr4**

### CODE REVIEW

File: db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js
Comment (blastshielddown) [inline db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js:1]: Think we also should add non-negative checks on these new columns. There are examples in other tables already

File: db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable("podcast", function (table) {
+      table.bigInteger("msat_total").notNullable().defaultTo(0);
+      table.check("msat_total >= 0", [], "msat_total_check");
```
Comment (blastshielddown) [inline db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js:5]: It'll help debug if these constraint names are unique
```suggestion
      table.check("msat_total >= 0", [], "podcast_msat_total_check");
```

File: db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    })
+    .alterTable("album", function (table) {
+      table.bigInteger("msat_total").notNullable().defaultTo(0);
+      table.check("msat_total >= 0", [], "msat_total_check");
```
Comment (blastshielddown) [inline db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js:9]:
```suggestion
      table.check("msat_total >= 0", [], "album_msat_total_check");
```

File: db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    });
+};
+
+exports.down = function (knex) {
+  return knex.schema
+    .alterTable("podcast", function (table) {
+      table.dropChecks(["msat_total_check"]);
```
Comment (blastshielddown) [inline db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js:16]:
```suggestion
      table.dropChecks(["podcast_msat_total_check"]);
```

File: db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      table.dropColumn("msat_total");
+    })
+    .alterTable("album", function (table) {
+      table.dropChecks(["msat_total_check"]);
```
Comment (blastshielddown) [inline db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js:20]:
```suggestion
      table.dropChecks(["album_msat_total_check"]);
```

**Comment by joshr4** [comment]
need double check the migration date before merging

**Review by blastshielddown — APPROVED** [review]

**Pull request merged**